src/lib.rs

```diff
@@ -12,12 +12,14 @@
 
 #[macro_use]
 extern crate derivative;
```

Comment:
nit: Extern crate syntax is no longer required, you can do `use derivative::Derivative` instead